doc/api/tls.md

```diff
@@ -1027,6 +1027,11 @@ added: v0.3.2
     force SSL version 3. The possible values depend on the version of OpenSSL
     installed in the environment and are defined in the constant
     [SSL_METHODS][].
+  * `secureOptions` {number} The options via bitmask affecting the protocol
```

Comment:
`A bitmask affecting the protocol....`

Please describe the default value/behaviour when not specified.